src/test/scala/za/co/absa/abris/avro/sql/CatalystAvroConversionSpec.scala

```diff
@@ -508,6 +508,21 @@ class CatalystAvroConversionSpec extends FlatSpec with Matchers with BeforeAndAf
     shouldEqualByData(dataFrame, result)
   }
 
+  it should "not print schema registry configs in the spark plan" in {
+    val sensitiveData = "username:password"
+    val schemaString = TestSchemas.NATIVE_SIMPLE_NESTED_SCHEMA
+
+    val fromAvroConfig = FromAvroConfig()
+      .withReaderSchema(schemaString)
+      .withSchemaRegistryConfig(Map(
+      AbrisConfig.SCHEMA_REGISTRY_URL -> dummyUrl,
+      "basic.auth.user.info" -> sensitiveData
+    ))
+
+    val column = from_avro('avroBytes, fromAvroConfig)
+    column.expr.toString() should not include sensitiveData
+  }
+
```

Comment:
Maybe this test would fit better into its own class (AvroDataToCatalystSpec)?
But feel free to keep it here. It's just an Idea.